_drafts/java-web-start.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Java Web Start
+categories: tech-issues
+---
+
+##Background
+
+Java Web Start was introduced in 2001 to allow applications to be launched through browsers or directly via the Java Network Launching Protocol (JNLP).
+
+Java Web Start Applications do not run inside the browser like Applets but still run in a restricted environment. Those restrictions can be removed by signing the applications with a trusted certificated. This has been encouraged since Java 7 update 21 to lower risks for desktop users and it has been re-inforced in follow-up updates.
+Two security changes to enhance authentication and authorization for Web Start (and Applets) were introduced back in January 2014 with Java 7 update 51.
+
+As of Java 7 update 51
+Rich Internet Applications must contain two things:
+1. Code signatures from a trusted authority.
```

Comment:
Looking at the rendered version via the "View" button:
https://github.com/jburel/blog/blob/webstart/_drafts/java-web-start.md
the itemization here still looks broken. Does this render correctly locally?